cloud/pkg/cloudhub/config/config.go

```diff
@@ -50,7 +50,7 @@ func InitConfigure(hub *v1alpha1.CloudHub) {
 			Config.Ca = ca
 			Config.CaKey = caKey
 		} else if !(ca == nil && caKey == nil) {
-			klog.Exit("Both of ca and caKey should be specified!")
+			panic("Both of ca and caKey should be specified!")
```

Comment:
Plz input an error value to `panic(..)` func

Reply:
Thanks, Fixed this